src/main/java/br/com/sbs/cubatech/category/CategoryRepository.java

```diff
@@ -42,6 +42,6 @@
         AND co.courseVisibility = 'PUBLIC'
         ORDER BY c.orderInSystem, s.orderInSystem        
         """)
-    List<Category> findCategoryBy();
+    List<Category> findCategoryByActiveWithSubCatgoryActiveAndCoursePublic();
```

Comment:
Tava faltando um `e` em `SubCategory`.

Mas fica uma sugestão:

```suggestion
    List<Category> findActiveCategoriesWithActiveSubCategoriesAndPublicCourses();
```